_about/notes/0.5.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
+pod-level proxy injection. Automatic injection doesn't work requires Kubernetes 1.9 or beyond and
+therefore doesn't work on older versions. TODO: Doc link?
+
+## Mixer adapters
+
+- **Open Policy Agent**. Mixer now has an authorization adapter implementing the [open policy agent](http://www.openpolicyagent.org) model,
+providing a flexible fine-grained access control mechanism. TODO: Doc link?
+
+- **RBAC**. Mixer now has a resource-based access control adapter. TBD: Doc link?
+
+- **Fluentd**. Mixer now has an adapter for [fluentd](https://www.fluentd.org) telemetry collection. TODO: Doc link?
+
+- **Stdio**. The stdio adapter now lets you log to files with support for log rotation & backup, along with a host
+of controls.
+
+## Security
+
+- **Bring Your Own CA**. There have been many enhancements to the 'bring your own CA' feature. TODO: Doc link?
+
+- **PKCS8**. Add support for PKCS8 keys to Istio PKI.
+
+## Other
+
+- **Component Logging**. Istio components now offer a rich set of command-line options to control local logging, including
+common support for log rotation. 
+
+- **Consistent Version Reporting**. Istio components now offer a consistent command-line interface to report their version information.
+
+- **Optional Instance Fields**. Within configuration, definitions of Mixer instances no longer need to include every field of the
+associated template. Omitted fields get a zero or empty value.
+
+
+TODO: What should we say about multi-cluster support in this release?
+TODO: What should we say about Grafana/dashboards in this release?
```

Comment:
Does this release cut include both the Mixer and Pilot dashboards in the grafana image?  If so, I think we should mention that we have initial versions of dashboards for monitoring the Istio components themselves.

Reply:
@ldemailly Who knows the answer to Doug's question?